libs/sdk-ui-charts/src/highcharts/typings/unsafe.ts

```diff
@@ -141,6 +141,8 @@ export interface IChartOptions {
     colorPalette?: IColorPalette;
     forceDisableDrillOnAxes?: boolean;
     verticalAlign?: ChartAlignTypes;
+    legendName?: string;
```

Comment:
What is the difference between legendName and legendLabel?

Reply:
Forgotten leftover from renaming, removed.